media-sound/amarok/amarok-9999.ebuild

```diff
@@ -111,11 +114,39 @@ pkg_postinst() {
 		echo "${1} ($(has_version ${1} || echo "not ")installed)"
 	}
 
+	optfeature "Audio CD support" kde-apps/audiocd-kio
+
 	if [[ -z "${REPLACING_VERSIONS}" ]]; then
-		elog "You'll have to configure amarok to use an external db server:"
-		use mariadb && elog "    $(pkg_is_installed dev-db/mariadb)" ||
+		elog "You'll have to configure amarok to use an external database server:"
+		if use mariadb ; then
+			elog "    $(pkg_is_installed dev-db/mariadb)"
+			elog "    For preliminary configuration of MariaDB Server please refer to"
+			elog "    https://wiki.gentoo.org/wiki/MariaDB#Configuration"
+		else
 			elog "    $(pkg_is_installed dev-db/mysql)"
+			elog "    For preliminary configuration of MySQL Server please refer to"
+			elog "    https://wiki.gentoo.org/wiki/MySQL#Configuration"
+		fi
 		elog "Please read https://community.kde.org/Amarok/Community/MySQL for details on how"
-		elog "to configure the external db and migrate your data from the embedded database."
+		elog "to configure the external database and migrate your data from the embedded database."
+		elog "To create external amarok database with default user/password please:"
+		elog "    1. Make sure that MySQL or MariaDB is installed"
+		elog "    2. If it's required make preliminary configuration of MySQL or MariaDB"
```

Comment:
`1. Make sure that MySQL or MariaDB is installed and configured (see above)`

Reply:
Messages are merged